pages/common/git-bug.md

```diff
@@ -0,0 +1,33 @@
+# git bug
+
+> A distributed bug tracker that uses git's internal storage, so no files are added in your project.
+> As you would do with commits and branches, you can push your bugs to the same git remote you are already using to collaborate with other people.
```

Comment:
Thank you for contribution, all examples look good to me. I think this sentence could be shortened but I cannot come up with a precise and shorter one.

Reply:
Hi, would you like it to be

> You may submit your issues to the same git remote you use to work with others, just like you would commits and branches.

Reply:
It seems to be more brief. Just go for it if you think this is precise enough.

Reply:
Remember to add a newline in the end of pages.

Reply:
Yeah sure, thanks for the suggestions.